11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/grid-layout/render-grid-in-template.md

```diff
@@ -25,5 +25,14 @@ You can create your own custom grid rendering files e.g for your favourite or cu
 or alternatively you can provide the path to where the file resides:
 
 ```csharp
-@Html.GetGridHtml(Model, "propertyAlias", "/views/mycustomrenderer.cshtml")
+@Html.GetGridHtml(Model, "propertyAlias", "mycustomrenderer")
```

Comment:
Taking away the path from this example, makes it identical to the example above.
Let's keep a path in this example - and then use one of the ones you've found to be working instead.

What do you think?

```suggestion
@Html.GetGridHtml(Model, "propertyAlias", "/Views/Partials/grid/mycustomrenderer.cshtml")
```

Reply:
@sofietoft 
The suggestion is good but this won't work.
when we add path in the `@Html.GetGridHtml(Model, "propertyAlias", "/Views/Partials/grid/mycustomrenderer.cshtml") ` this will cause issue and trying to find view in /grid/ and added path in the code block. so this won't work.

Reply:
Ah, alright. So is there are path we can add here that works?
Or is it because adding a path in this code doesn't work? 🤔

Reply:
If it doesn't work to include the path in this code, it should be taken out.
I don't think it makes sense to have the same code example listed twice.
I also already mentions in line 19 where you should place the file, right?

Perhaps the only extra thing we need here, is the list of other potential places to put the file?